front50-migrations/src/main/java/com/netflix/spinnaker/front50/migrations/MigrationRunner.java

```diff
@@ -22,10 +22,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Component;
 
+/**
+ * Migration runner runs all the registered migrations as scheduled. By default migration runner
+ * will <strong>not</strong> run and need to set <strong>`migrations.enabled`</strong> property to
+ * enable it.
+ *
+ * <p>Note: Ideally migrations should be running only on one instance.
+ */
 @Component
+@ConditionalOnProperty(name = "migrations.enabled", havingValue = "true", matchIfMissing = false)
```

Comment:
this is reverse behavior from what is there today (default not running). I think this makes sense, and I doubt someone has additional migrations on top of what's already in front50

Reply:
There are very few migrations that are active now and some of them have conditional on them directly so on the OSS side we will turn off those. May be better leave it `default running` and turn off explicitly ?